functions/Get-DbaLogin.ps1

```diff
@@ -260,7 +260,7 @@ function Get-DbaLogin {
 
                 if ($Detailed) {
                     $loginName = $serverLogin.name
-                    $query = $loginProperty.Replace('<#LoginName#>', "$loginName")
+                    $query = $loginProperty.Replace("''", "$loginName")
```

Comment:
There is a reason that is in the statement for replace:

https://github.com/sqlcollaborative/dbatools/blob/e9cefc46343fe220f3129eeee5c9420c0bacd641/functions/Get-DbaLogin.ps1#L176-L183

![image](https://user-images.githubusercontent.com/11204251/67585877-0a8f1080-f716-11e9-9466-ff251130dced.png)

Reply:
We need to know if the culture is the reason this is causing issues or not. In the issue it is noted `de-AT` was the OP's culture.

Reply:
I get the same issue, my culture is UK/en-GB.

Reply:
I think it needs to be either
```
                    $query = $loginProperty.Replace("'<#LoginName#>'", "$loginName")```
or
                   ``` $query = $loginProperty.Replace("''", "$loginName")```

but 
$query = $loginProperty.Replace("<#LoginName#>", "$loginName") doesn't seem valid and it does fail for me in that place with -Detailed

Reply:
That I can't explain but it works perfectly fine for my system...I added a debug output so the query shows it is resolving properly. I receive no errors.
![image](https://user-images.githubusercontent.com/11204251/67586950-6f4b6a80-f718-11e9-96ae-48c546f5bb46.png)

Reply:
Also note that our test pass for this command as well, and we do test that the detailed output is showing up.

Reply:
odd, let me triple check.

Reply:
![image](https://user-images.githubusercontent.com/31617434/67587342-90797e80-f74b-11e9-86c7-b247d890c935.png)

Reply:
So this is an issue as to why replace is not seeing `<#loginname#>'`, just replacing it to be `''` because that is what the error shows is not sufficient for me to approve this PR.